docs/examples/test_examples.py

```diff
@@ -0,0 +1,268 @@
+""" Tests for the files in nameko/docs/examples
```

Comment:
probably wouldn't be a bad thing to always run (e.g. to make sure examples work for all versions we support)